lib/membrane_live_audio_mixer.ex

```diff
@@ -2,8 +2,8 @@ defmodule Membrane.LiveAudioMixer do
   @moduledoc """
   This element performs audio mixing for live streams.
 
-  Live Audio Mixer starts to mix audio after the first input pad is added,
-  from this point, the mixer will produce a audio until element termination.
+  Live Audio Mixer starts to mix audio after the first input pad is added or (in manual mode) when `start_mixing` notification is send.
```

Comment:
This is not exactly scope of this PR, but while checking docs I noticed `LiveAudioMixer` and queues modules are not grouped and nested by the prefix, like the rest of modules are.